### PR TITLE
cleanup cloudformation

### PR DESCRIPTION
**Pull request by fabiocicerchia**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -34,7 +32,7 @@ APP_XDEBUG_HOST=docker.for.mac.localhost
 
 ###> snc/redis-bundle ###
 # passwords that contain special characters (@, %, :, +) must be urlencoded
-REDIS_URL=redis://redis
+REDIS_HOST=redis
```
Comment (JellyBellyDev) [inline .env:35]: Is already present in line 17. why this change?

Comment (drAlberT) [reply .env:35]: just remove REDIS_URL

**Review by drAlberT — CHANGES_REQUESTED** [review]

**Review by drAlberT — APPROVED** [review]

**Review by JellyBellyDev — APPROVED** [review]
LGTM  👍
